src/AppBundle/Controller/DefaultController.php

```diff
@@ -1,19 +0,0 @@
-<?php
-
-namespace AppBundle\Controller;
-
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
-
-class DefaultController extends Controller
```

Comment:
I removed this controller as it is not really useful and would be complicated to manage with many repositories.